scrunch/datasets.py

```diff
@@ -992,6 +1026,19 @@ def settings(self, _):
         # Protect the `settings` property from external modifications.
         raise TypeError('Unsupported operation on the settings property')
 
+    @property
+    def filters(self):
+        _filters = {}
+        for f in self.resource.filters.index.values():
+            filter_inst = Filter(f)
+            _filters[filter_inst.name] = filter_inst
+        return _filters
+
+    @filters.setter
+    def filters(self, _):
+        # Protect the `settings` property from external modifications.
+        raise TypeError('Use add_filter method or edit_filter to modify filters')
```

Comment:
Right, but the wording here suggest that there is an edit_filter method in the Dataset, which we don't